ui-components/src/forms/FieldGroup.tsx

```diff
@@ -59,7 +61,7 @@ const FieldGroup = ({
               defaultChecked={item.defaultChecked || false}
               ref={register(validation)}
             />
-            <label htmlFor={item.id} className="font-semibold">
+            <label htmlFor={item.id} className={`font-semibold ${fieldLabelClassName}`}>
```

Comment:
@emilyjablonski , can you update the storybook story for this to show this new option?

Reply:
We don't have any stories for this component, I can add the file 👍